src/transformers/models/layoutlmv2/layoutlmv2_fast.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
```

Comment:
Don't think you need this `layoutlmv2_fast.py` file, as files for fast tokenizers are typically called `tokenization_layoulmv2_fast.py`, and I see this PR already has this file.